src/PactNet.Native/PactExtensions.cs

```diff
@@ -51,17 +51,51 @@ public static IPactBuilderV3 UsingNativeBackend(this IPactV3 pact, int? port = n
         }
 
         /// <summary>
-        /// Initialise a new pact on the server with the correct version
+        /// Establish a new messagePact using the native backend
+        /// </summary>
+        /// <param name="messagePact">Pact details</param>
+        /// <returns>Pact builder</returns>
+        /// <remarks>
+        /// If multiple mock servers are started at the same time, you must make sure you don't supply the same port twice.
```

Comment:
I don't think these docs make sense for message pacts do they? I don't think it starts a mock server at all.